api/envoy/config/core/v3/base.proto

```diff
@@ -165,6 +167,10 @@ message Node {
   // directly to the management server.
   google.protobuf.Struct metadata = 3;
 
+  // Map from xDS resource type URL to dynamic context parameters. These may
+  // vary at runtime (unlike other fields in this message).
+  map<string, xds.core.v3.ContextParams> dynamic_parameters = 12;
```

Comment:
Will dynamic_parameters only passed from envoy in the request, or will they also be returned in the response? 
In other words, will envoy be able to tell that response is based on the passed parameters?

Reply:
In the future `xdstp://` implementation, yes, but for `Node` this is not currently planned, since the use of `Node` here is a legacy concession right now.

I think for your use case, having this in the response won't help a lot, since it's not enough to know that a response has been received, you need to know the configuration has taken effect on a per-resource basis. You need something like the `xdstp://` naming scheme to know that each of the new clusters (for example) are warmed and available.